// 2019.12.02 이현경
package c_control;

import java.io.IOException;

public class Main {
  public static void main(String[] args) throws IOException {
	  new WhileExam3(); // 다른 클래스여도 간은 패키지 안에 있기 때문에 'Exam1'이 실행됨(익명형 - 객체명이 없다.)
  }
}
